src/components/retrieve_from_weaviate/requirements.txt

```diff
@@ -0,0 +1,5 @@
+llama-index==0.8.68
+weaviate-client
+transformers
+torch
```

Comment:
We should fix the version of all our dependencies.

Reply:
Transformers and torch are now useless since we now embed using the embed_text component. I've settled the version of the weaviate-client.